src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -112,25 +118,62 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
         .backCallback=${this._backTapped}
         .tabs=${configSections.automations}
       >
-        <ha-button-menu
-          corner="BOTTOM_START"
-          slot="toolbar-icon"
-          @action=${this._handleMenuAction}
-          activatable
-        >
+        <ha-button-menu corner="BOTTOM_START" slot="toolbar-icon">
           <ha-icon-button
             slot="trigger"
             .label=${this.hass.localize("ui.common.menu")}
             .path=${mdiDotsVertical}
           ></ha-icon-button>
 
           <mwc-list-item
-            aria-label=${this.hass.localize(
-              "ui.panel.config.automation.editor.edit_ui"
-            )}
             graphic="icon"
-            ?activated=${this._mode === "gui"}
+            .disabled=${!stateObj}
+            @click=${this._runActions}
+          >
+            ${this.hass.localize("ui.panel.config.automation.editor.run")}
+            <ha-svg-icon slot="graphic" .path=${mdiPlay}></ha-svg-icon>
+          </mwc-list-item>
+
+          ${stateObj
+            ? html`<a
+                href="/config/automation/trace/${this._config
+                  ? this._config.id
+                  : ""}"
+                target="_blank"
+                .disabled=${!stateObj}
+              >
+                <mwc-list-item graphic="icon" .disabled=${!stateObj}>
+                  ${this.hass.localize(
+                    "ui.panel.config.automation.editor.show_trace"
```

Comment:
This becomes "Show Trace", but I think that it should be "Traces" to be consistent.

Reply:
👍 Done